tests/functional/firefox/test_all.py

```diff
@@ -64,39 +64,22 @@ def test_firefox_esr(base_url, selenium):
     product = page.select_product('Firefox Extended Support Release')
     product.select_platform('Linux 32-bit')
     product.select_language('English (US)')
-    assert page.is_download_button_displayed
-    assert page.is_download_link_valid
-    assert 'product=firefox-esr-latest-ssl' and 'os=linux' and 'lang=en-US' in page.download_link
+    assert page.is_desktop_download_button_displayed
+    assert page.is_desktop_download_link_valid
+    assert 'product=firefox-esr-latest-ssl' and 'os=linux' and 'lang=en-US' in page.desktop_download_link
 
 
 @pytest.mark.smoke
 @pytest.mark.nondestructive
 def test_firefox_android(base_url, selenium):
     page = FirefoxAllPage(selenium, base_url).open()
-    product = page.select_product('Firefox Android')
-    product.select_platform('Intel devices (Android 4.1+ x86 CPU)')
-    assert page.is_download_button_displayed
-    assert page.is_download_link_valid
-    assert 'product=fennec-latest' and 'os=android-x86' and 'lang=multi' in page.download_link
+    page.select_product('Firefox Android')
+    assert page.is_android_download_button_displayed
 
 
 @pytest.mark.smoke
 @pytest.mark.nondestructive
-def test_firefox_android_beta(base_url, selenium):
```

Comment:
Can the tests here for Android Beta / Nightly be updated now too?

Reply:
✅